src/main/java/chess/Board.java

```diff
@@ -1,81 +1,151 @@
 package chess;
 
-import pieces.Pawn;
-
+import pieces.Piece;
+import static utils.StringUtils.*;
 import java.util.ArrayList;
 import java.util.List;
-
-
 public class Board {
 
-    private List<Pawn> whitePawnList = new ArrayList<>();
-    private List<Pawn> blackPawnList = new ArrayList<>();
+    private List<Piece> whitePieceList = new ArrayList<>();
+    private List<Piece> blackPieceList = new ArrayList<>();
 
-    private String startLine = "* * * * * * * *";
+    private List<Piece> whitePawnList = new ArrayList<>();
+    private List<Piece> blackPawnList = new ArrayList<>();
 
-    private final int PAWNCOUNT = 8;
+    private String startLine = "********";
 
-    public Pawn findPawn(Pawn.Color color, int index) {
-        if (color == Pawn.Color.WHITE) {
+    private final int PAWN_COUNT = 8;
+
+    public Piece findPawn(Piece.Color color, int index) {
+        if (color == Piece.Color.WHITE) {
             return whitePawnList.get(index);
         }
         return blackPawnList.get(index);
     }
 
-    public void add(Pawn pawn) {
-        if (pawn.getColor() == Pawn.Color.WHITE) {
+    public Piece findPiece(Piece.Color color, int index){
+        if(color == Piece.Color.WHITE){
+            return whitePieceList.get(index);
+        }else {
```

Comment:
중괄호와 그 다음 키워드 사이는 한 칸 띄우는 것이 컨벤션이고요.

```suggestion
        } else {
```

`else` 없이도 `return` 을 하면 메소드의 실행이 종료될 것 같네요. `if` 분기에 해당했다면요.

Reply:
@hiro032 여기 개선 제안 사항이 반영되지 않았네요.

Reply:
> @hiro032 여기 개선 제안 사항이 반영되지 않았네요.

수정하다보니 else자체가 필요없는거같아서 제거했습니다.